<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="dirLineEdit"/>
     </item>
     <item>
      <widget class="QListWidget" name="fileList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="chooseDirButton">
       <property name="text">
        <string>Choose Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createDirButton">
       <property name="text">
        <string>Create Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dirExistsButton">
       <property name="text">
        <string>Dir exists?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dirOrFileButton">
       <property name="text">
        <string>Dir or file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="folderContentsButton">
       <property name="text">
        <string>Folder Content</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
